llms/mlx_lm/tuner/linear.py

```diff
@@ -0,0 +1,88 @@
+import math
```

Comment:
I feel like the name of this file should be a bit more descriptive. Maybe `lora_layers.py` or even `lora.py`?

Reply:
either of them makes sense to me, I will update it.